scanapi/templates/markdown.jinja

```diff
@@ -2,38 +2,56 @@
 
 {% for response in responses %}
 {% set request = response.request %}
-## Request: {{ request.method }} {{ request.url }}
+## {{ request.method }} {{ request.url }}
 
-HEADERS:
-<details><summary></summary><p>
+### Request
 
-```
-{{ request.headers }}
-```
-</p></details>
+#### Headers
 
-### Response: {{ response.status_code }}
+{% if request.headers|length > 0 %}
+| Header    | Value       |
+|-----------|-------------|
+{% for key, value in request.headers.items() %}
+| {{ key }} | {{ value }} |
+{% endfor %}
+{% else %}
+None
+{% endif %}
 
-Is redirect? {{ response.is_redirect }}
+### Response
 
-HEADERS:
-<details><summary></summary><p>
+#### Metadata
 
-```
-{{ response.headers }}
-```
-</p></details>
+|             |                            |
+|-------------|----------------------------|
+| status code | {{ response.status_code }} |
+| redirect    | {{ response.is_redirect }} |
```

Comment:
I like the idea of having tables for HTTP headers but I'm not so sure yet if this is the best way to display the pieces of information above.

Reply:
I like the tables too. I think it is a good way. On Postman, for example, it is a table too

![image](https://user-images.githubusercontent.com/2728804/71739405-198def80-2e38-11ea-9622-c5fe63dc07e2.png)